Makefile

```diff
@@ -99,32 +88,33 @@ release-binaries: ## Build release binaries
 test: generate verify ## Run tests
 	bazel test --nosandbox_debug //pkg/... //cmd/... $(BAZEL_ARGS)
 
+.PHONY: install-dev
```

Comment:
should this be `install-dev-binaries` instead? It might be a good idea to add dependencies here on the `clusterctl` and `clusterawsadm` binaries to ensure that it is installing an up to date build.

Reply:
we are using -dev as a suffix, so I'd recommend `install-binaries-dev` (like `docker-push-dev`)